### ISSUE TITLE
Need documentation on how to move to selenium 3.0

### ISSUE THREAD

**Issue by shankarkc** [groupon/Selenium-Grid-Extras#280]
Selenium is moved to 3.0 version Its not clear how selenium Grid extra can be used to work with it. Please provide some documentation explaining the changed required. i just changed webdriver version to 3.0.1 But the changes on the hub getting reverted and falls back to 2.53.1 Pls let us know how to use selenium 3.0

**Comment by smccarthy** [comment]
@shankarkc Do you see any errors? I tested upgrading from 2.53.1 to 3.0.0 when it was released, and it seemed to work OK. The main change I had to make, was to change the node config files. The node config file changed slightly, so every time Grid Extras starts, I check if you are using selenium 3.X and the selenium 2.X node config file, then I update the config file to be 3.X compliant. The same if you downgrade from 3.X to 2.X, I check the node config file to see if it needs to be updated.

I don't know of any reverting logic and falling back to 2.53.1.

Is there any logs you can provide, as well as the version of selenium-grid-extras, and the selenium-grid-extras config file, node config file, hub config file ?

Thank you.

**Comment by shankarkc** [comment]
Sorry Shawn,
It was my mistake. I ran the code unknowingly on a machine where selenium grid extra was insalled using chef. Chef use to revert my changes. I used another aws machine and noticed that..
1. We just need to change webdriver version to 3.0.1
2. Install JDK 1.8( without it i got error minor version XXXXX not supported. After installing JDK 1.8 it worked fine.
I will close the ticket.
Thanks
Shankar KC
